src/javaRosa.js

```diff
@@ -44,7 +44,8 @@ define([
             'default', 'short', 'long', 'audio', 'video', 'image'
         ],
         _nextItextItemKey = 1,
-        NO_MARKDOWN_MUGS = ['Item', 'Group', 'FieldList', 'Repeat'];
+        NO_MARKDOWN_MUGS = ['Item', 'Group', 'FieldList', 'Repeat'],
+        EXPERIMENTAL_UI;
```

Comment:
https://github.com/dimagi/Vellum/pull/493/files#diff-b860087dfb0ae7f09a955a95f517a309R298

For things like xpath I have them using single line right now so that they show up as they have been before. 

I think it may make sense to split up richtext to richsingeline/richtextarea. single line would be more for display like xpaths are now/the draggable question widget. where as textarea is where you do most of your editing like the label or advanced expression editor